cirq-core/cirq/circuits/circuit.py

```diff
@@ -1169,6 +1172,10 @@ def to_text_diagram_drawer(
             The TextDiagramDrawer instance.
         """
         qubits = ops.QubitOrder.as_qubit_order(qubit_order).order_for(self.all_qubits())
+        draw_cregs = bool(protocols.control_keys(self))
+        if draw_cregs:
+            cbits = tuple(protocols.measurement_key_objs(self))
```

Comment:
Shouldn't this be `control_keys`? We don't want to draw lines for all the measurement keys, just the ones that correspond to a control key.